src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -224,7 +249,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
      */
     protected function createConfigurationDefinition()
     {
-        return new FixerConfigurationResolverRootless('order', [
+        return new FixerConfigurationResolver([
```

Comment:
wow wow wow wow wow, what is going on here? it is a BC breaker

Reply:
I thought so. Could you please tell me, how else should I do that?

Reply:
revert that one line (and removal of tests), and everything will work how you want without breaking BC

Reply:
You're right. It confused me that there is no single root (order) anymore, since there are two now (order and sortAlgorithm).